rhel7/profiles/cis.profile

```diff
@@ -3,9 +3,11 @@ documentation_complete: true
 title: 'CIS Red Hat Enterprise Linux 7 Benchmark'
 
 description: |-
-    This baseline aligns to the Center for Internet Security
-    Red Hat Enterprise Linux 7 Benchmark, v2.2.0, released
-    12-27-2017.
+    This profile defines a baseline that aligns to the Center for Internet Security®
+    Red Hat Enterprise Linux 7 Benchmark, v2.2.0, released 12-27-2017.
```

Comment:
Is this date and benchmark version accurate?

Reply:
They are correct.
But I just found out there is a 2.2.0.1, from Oct 29th 2018, :/

Reply:
v2.2.0.1 seems to be unavailable for download.
And support discussion mentions there is virtually no difference between 2.2.0 and 2.2.0.1